app/api/search/search.js

```diff
@@ -650,7 +650,9 @@ const instanceSearch = elasticIndex => ({
     // queryBuilder.query() is the actual call
     return elastic
       .search({ index: elasticIndex || getCurrentTenantIndex(), body: queryBuilder.query() })
-      .then(response => processResponse(response, templates, dictionaries, language, query.filters))
+      .then(response => {
+        return processResponse(response, templates, dictionaries, language, query.filters);
+      })
```

Comment:
This is yielding a warning, and it looks like it was intercepted to do some debugging, but it should be reverted to the arrow function directly returning the processResponse as the warning suggests.